clarity_epp.py

```diff
@@ -94,10 +94,17 @@ def export_merge_file(args):
 
 
 def export_removed_samples(args):
-    """Export removed sampels table."""
+    """Export removed samples table."""
     clarity_epp.export.sample.removed_samples(lims, args.output_file)
 
 
+def export_sample_indications(args):
```

Comment:
Is there a reason to provide all args instead of the specific arguments ? 
`export_sample_indications(output_file, artifact_name, sequencing_run, sequencing_run_project)`
Does the argument `args`only contain the sub parser arguments?

Reply:
Yes and yes.

We need to this because we use this pattern:
```python
parser_export_sample_indications.set_defaults(func=export_sample_indications)
args.func(args)
```